### PR TITLE
Rename "Analysis" to "Analyse" in German menu (Alignment to page name)

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR renames "Analysis" to "Analyse" in the German menu of the page.

Analysis is a German word, but it is not correct in this context, see https://www.duden.de/rechtschreibung/Analysis:

> Teil der Mathematik, in dem mit Grenzwerten gearbeitet, die Infinitesimalrechnung angewendet wird

OR

> Voruntersuchung beim Lösen geometrischer Aufgaben

I tested this change locally & everything was still correct. 

---

Here is a preview of this change:

Unknown

### CODE REVIEW

**Comment by MikeMcC399** [comment]
@Ein-Tim 

- The naming has been confusing from the beginning and it was agreed in https://github.com/corona-warn-app/cwa-website/issues/1776 to make it an English language word like the other top menu items.

It's unfortunate that the English and German words are so similar but not exactly the same.

The alternative design change would be to rename all the menu items to German when displaying any German page e.g. instead of "Science" write "Wissenschaft". At this point in time that would be quite disruptive though.

In short, I recommend no change to the menu.

**Comment by Ein-Tim** [comment]
@MikeMcC399 

I disagree here, it is OK to have menu items "Science" or "Community" as they are widely known and do not have another meaning in Germany.

This is **not** the case for "Analysis" which has a different meaning in German than in English. 

Also, after clicking on "Analysis", the page already shows "Analyse" at the top, so this is only an alignment of the menu item to the name of the page.

**Comment by Ein-Tim** [comment]
I have changed the title of this PR to make clearer that this is an alignment and nothing else.

**Comment by MikeMcC399** [comment]
@Ein-Tim 

- I have provided an alternative suggestion to resolve the problem you highlighted in this PR. The new issue is https://github.com/corona-warn-app/cwa-website/issues/2352 which does not involve changing the language of a menu item, nor does it involve any formatting changes, so it is very easy to implement. You are welcome to comment there.

**Comment by Ein-Tim** [comment]
Closing in favour of #2390.
